### DISCUSSION TITLE
Use Stratis as shared storage between 3 hosts

### DISCUSSION THREAD

**Discussion by mehrdad2000** [stratis-storage/project#594] [Q&A]
Hi
I have 3 server that work together as cluster.
Need share storage that replicate a volume between them.

First of all i use nfs but it has performance issue in high i/o.
Second use redhat glusterfs but eol of this product is near.
Third solution is use ceph and minio but i heard configuration and maintaining of them is though.

Any idea? Must be opensource, simple, work in high i/o, redhat supported.

Is it possible to use Stratis in this scenario?

**Comment by jbaublitz** [comment]
@mauelsha has done some work on stratisd in a clustered environment, but I'm unsure of the status of that at the moment. Outside of that, NFS is probably the best way to handle multinode Stratis at the moment. We recently worked on supporting NFS on top of Stratis filesystems so that should be working now.

**Comment by mehrdad2000** [comment]
@jbaublitz do you have tuning on nfs? Because i have performance issue on nfs when disk i/o increased.

**Comment by jbaublitz** [comment]
What kind of performance problems are you seeing? Is this NFS on top of a simple XFS filesystem or a more complicated stack?

**Comment by mehrdad2000** [comment]
NFS on top of a simple XFS filesystem

**Reply by jbaublitz** [reply]
Stratis is an XFS based storage solution. Unless there is some other way of sharing XFS across nodes, NFS is probably the best candidate for this use case. If you have performance concerns, that would probably be something to take up with NFS.
